auth-api/tests/unit/api/test_org.py

```diff
@@ -1489,7 +1490,17 @@ def test_approve_org_with_pending_affidavits(client, jwt, session, keycloak_mock
                                content_type='application/json')
     assert org_response.status_code == http_status.HTTP_201_CREATED
 
-    tasks = TaskService.fetch_tasks(task_status=[TaskStatus.OPEN.value], page=1, limit=10)
+    task_search = TaskSearch(
```

Comment:
is it possible to make some of these optional? and then used named parameters for the ones that exist?

Reply:
Ex. TaskSearch(
param1=[TaskStatus.OPEN.value],
)
etc.

Reply:
done.